pgbouncer-mixin/alerts.libsonnet

```
@@ -0,0 +1,92 @@
+{
+  new(this): {
+
+    groups: [
+      {
+        name: this.config.uid,
+        rules: [
+          {
+            alert: 'HighNumberClientWaitingConnections',
+            expr: |||
+              pgbouncer_pools_client_waiting_connections > %(alertsHighClientWaitingConnections)s
+            ||| % this.config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'May indicate a bottleneck in connection pooling, where too many clients are waiting for available server connections,',
```

I don't think you need a comma after `pooling`
```suggestion
              summary: 'May indicate a bottleneck in connection pooling where too many clients are waiting for available server connections.',
```